scripts/snapshot_stats.py

```diff
@@ -0,0 +1,430 @@
+import os
+import argparse
+from datetime import datetime
+
+import jinja2
+from peewee import SqliteDatabase, DatabaseError
+from playhouse.reflection import (
+    generate_models,
+)
+from pandas import DataFrame
+
+
+"""
+Create a CanDIG-Server DataBase Snapshot Report.
+
+usage: python snapshot_stats.py [-h] [--markdown] [--html]
+                       [--destination /output/directory/]
+                       database
+
+positional arguments:
+  database              Path do CanDIG-Server database file
+
+optional arguments (at ):
+  -h, --help            show this help message and exit
+  --destination /output/directory/
+                        Directory where the outputs will be saved
+  (At least of the below arguments must be used)
```

Comment:
At least one..?

Reply:
If you don't pass neither `--html` nor `--markdown` it will not generate the report, as dicussed on slack.